### DISCUSSION TITLE
Mise à jour

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#21] [General]
Bonsoir,
Quand on voit que vous avez fait une mise à jour, par exemple là il y a 4 heures.
Est ce qu'on peut synchroniser avec votre site tout de suite ou doit on attendre ?

Merci par avance

**Comment by pretrehr** [comment]
Bonjour,
Vous pouvez synchroniser lorsque vous le souhaitez.
Lorsqu'une version est fonctionnelle, ce badge apparaît dans le README.md :
![image](https://user-images.githubusercontent.com/43851883/107886343-801c8280-6eff-11eb-8cad-5a13efe56587.png)
Donc quand vous voyez qu'une mise à jour est disponible et que ce badge est présent, vous pouvez directement la récupérer sans trop vous poser de questions.
